packages/react/src/component-tests/IcToast/IcToast.cy.tsx

```diff
@@ -125,31 +124,31 @@ describe("IcToast Visual Regression and A11y Testing", () => {
       "slotted-button",
       DEFAULT_TEST_THRESHOLD + 0.02
     );
-    cy.wait(150).checkA11y(undefined, CYPRESS_AXE_OPTIONS);
+    cy.wait(150).checkA11y;
   });
 
   it("renders slotted link", () => {
     mount(<SlottedLinkToast />);
     cy.get("ic-button").click();
 
     cy.wait(100).compareSnapshot("slotted-link", DEFAULT_TEST_THRESHOLD + 0.02);
-    cy.wait(100).checkA11y(undefined, CYPRESS_AXE_OPTIONS);
+    cy.wait(100).checkA11y;
```

Comment:
What makes cy.wait(100).checka11y different to cy.checkA11yWithWait(undefined, 100) ?